src/components/layout/dashboardLayout.vue

```diff
@@ -11,30 +11,40 @@ import "animate.css";
 import { useRouter, useRoute } from "vue-router";
 import { useUserProfile } from "@/stores/profile";
 import { useStore } from "@/stores/user";
+import { storeToRefs } from "pinia";
+import { useMessageStore } from "@/stores/message";
 import ChatWidget from "@/components/ui/Message/ChatWidget.vue";
 
-let store = useStore();
-const accountType = computed(() => {
-  return store.getUser.data.user.type;
-});
-onMounted(() => {
-  return accountType;
-});
-
+const messageStore = useMessageStore();
 const profileStore = useUserProfile();
+const messageLength = ref()
+const messageNum = ref()
 const router = useRouter();
 const route = useRoute()
 const closeNav = ref(false);
 const closeBackdrop = ref(false);
 const showDropdown = ref(false);
 const showNotificationDropdown = ref(false);
 const showChatWidget = computed(() => !route.meta.hideChatWidget);
+const receivedMessages = ref([])
+const { allMessages } = storeToRefs(messageStore)
 const userDetails = computed(() => {
   return profileStore?.user?.data;
 });
 
 const props = defineProps(['height', 'overflow'])
 
+let store = useStore();
+const accountType = computed(() => {
+  return store.getUser.data.user.type;
```

Comment:
Use optional chaining here incase the data doesn't exist

store.getUser?.data?.user?.type